RELEASE_NOTES

```diff
@@ -3,9 +3,46 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
-The latest version of easybuild-easyconfig provides 4,596 easyconfig files, for 737 different software packages,
+The latest version of easybuild-easyconfig provides 4,818 easyconfig files, for 778 different software packages,
 34 different (compiler) toolchains, 12 software bundles and 1 meta-package.
 
+v2.5.0 (December 17th 2015)
+---------------------------
+
+feature + bugfix release
+- add GCCcore easyconfig that can be used as base for all compilers (without getting in the way) (#2214)
+    - + easyconfig for GCC/4.9.3-2.25: bundle of GCCcore 4.9.3 and binutils 2.25
```

Comment:
maybe add that this is the replacement of the `GNU` toolchain? It now loads a fully working toolchain.